modules/light-clients/08-wasm/keeper/keeper.go

```diff
@@ -118,7 +118,7 @@ func (Keeper) storeWasmCode(ctx sdk.Context, code []byte) ([]byte, error) {
 
 	// create the code in the vm
 	ctx.GasMeter().ConsumeGas(types.VMGasRegister.CompileCosts(len(code)), "Compiling wasm bytecode")
-	vmChecksum, err := ibcwasm.GetVM().StoreCode(code)
+	vmChecksum, err := ibcwasm.GetVM().StoreCodeUnchecked(code)
```

Comment:
I think we need to do either `StoreCode` or `StoreCodeUnchecked` depending on where this keeper `storeWasmCode` is being called from. There are two situations:

- keeper `storeWasmCode` is [called in `InitGenesis`](https://github.com/cosmos/ibc-go/blob/2bd29c08fd1fe50b461fc33a25735aa792dc896e/modules/light-clients/08-wasm/keeper/genesis.go#L14) -> We should call `StoreCodeUnchecked`.
- keeper `storeWasmCode` is [called in `StoreCode` RPC handler](https://github.com/cosmos/ibc-go/blob/2bd29c08fd1fe50b461fc33a25735aa792dc896e/modules/light-clients/08-wasm/keeper/msg_server.go#L25) -> We should call `StoreCode`.